src/VisualStudio/Core/Test/Completion/CSharpCompletionSnippetNoteTests.vb

```diff
@@ -21,17 +21,10 @@ class C
     void M() { }
 }]]></document>
 
-        Public Shared ReadOnly Property AllCompletionImplementations() As IEnumerable(Of Object())
-            Get
-                Return TestStateFactory.GetAllCompletionImplementations()
-            End Get
-        End Property
-
         <WorkItem(726497, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/726497")>
-        <MemberData(NameOf(AllCompletionImplementations))>
         <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
Needs to be converted back to WpfFact (applies for rest of file.)

Reply:
Note: xunit analyzers do not support VB, which is really risky in this case. Since this is just a symbol analyzer, you may want to consider adding VB support to the analyzer, building it locally to catch further offenses, and of course send the fix upstream.

Reply:
Xunit rejected our offer to add VB support. Will discuss forking the analyzer project into roslyn-analyzers for use in this repository with VB support.